### PR TITLE
Removing OpenDistro tests

### PR DESCRIPTION
**Pull request by VachaShah**
Signed-off-by: Vacha Shah <vachshah@amazon.com>

### Description
Removing ODFE tests since OpenDistro is now archived

### Issues Resolved
_List any issues this PR will resolve, e.g. Closes [...]._ 

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

**Review by dblock — DISMISSED** [review]

File: Makefile [hunk 1]
```diff
@@ -152,16 +152,6 @@ godoc: ## Display documentation for the package
...
-	docker-compose --project-directory .ci/opendistro build;
-
-cluster.opendistro.start:
-	docker-compose --project-directory .ci/opendistro up -d ;
-	sleep 20;
-
-cluster.opendistro.stop:
-	docker-compose --project-directory .ci/opendistro down ;
-
 cluster.opensearch.build:
```
Comment (VijayanB) [inline Makefile:155]: nit : May be we can call it as cluster.build?

Comment (VachaShah) [reply Makefile]: Done.

**Review by VijayanB — APPROVED** [review]

**Comment by VachaShah** [comment]
Tests against OpenSearch main are failing for all PRs. Created #163

**Comment by VachaShah** [comment]
@dblock @VijayanB Can I get approvals again?  The earlier approvals got dismissed after a rebase. The tests against OpenSearch main are fixed and the CI is passing now. Thank you!

**Review by VijayanB — APPROVED** [review]

**Review by saratvemulapalli — APPROVED** [review]

**Pull request merged**